core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -153,7 +152,7 @@
       it "generates the correct total amount" do
         order.cancellations.short_ship([inventory_unit_1])
         order.cancellations.short_ship([inventory_unit_2])
-        expect(line_item.adjustments.non_tax.sum(:amount)).to eq(-1.67)
+        expect(line_item.adjustments.map(&:amount)).to match_array([0.01, -0.83, -0.84])
```

Comment:
This is a little strange, as the three values now add up to -1.66. Is there a tax adjustment hidden there?